config/packages/security.yaml

```diff
@@ -0,0 +1,37 @@
+security:
+    # https://symfony.com/doc/current/security.html#where-do-users-come-from-user-providers
+    providers:
+        users_in_memory: { memory: null }
+        users:
+            entity:
+                class: 'App\Entity\User'
+                property: 'username'
+    encoders:
+        Symfony\Component\Security\Core\User\UserInterface:
+            algorithm: bcrypt
```

Comment:
https://symfony.com/blog/new-in-symfony-4-3-native-password-encoder
`algorithm: auto`

Dann sucht sich Symfony automatisch das Beste. z.B. Sodium ^^
Ich meine sogar, dass Symfony sogar dann vorhandene Passwörter bei Logins auch migriert